sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/Tables/TablesTestEnvironment.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+
+namespace Microsoft.Azure.WebJobs.Host.FunctionalTests
+{
+    public class TablesTestEnvironment: TestEnvironment
```

Comment:
Doesn't this already exist from Tables test project?

Reply:
It does but I'm not sure I want to source include it. Wanted to keep projects decoupled.